python.json

```diff
@@ -94,5 +94,19 @@
   {"id": "py.pydub", "imports": ["pydub"], "name": "Pydub", "repo": "jiaaro/pydub", "tags": ["toolkit"], "tech": ["audio"]},
   {"id": "py.pyechonest", "imports": ["pyechonest"], "name": "Pyechonest", "repo": "echonest/pyechonest", "tags": ["music-search"], "tech": ["api"]},
   {"id": "py.timeside", "imports": ["timeside"], "name": " TimeSide", "repo": "Parisson/TimeSide", "tags": ["web-audio"], "tech": ["audio"]},
-  {"id": "py.tinytag", "imports": ["tinytag"], "name": "tinytag", "repo": "devsnd/tinytag", "tags": ["metadata"], "tech": ["audio"]}
+  {"id": "py.tinytag", "imports": ["tinytag"], "name": "tinytag", "repo": "devsnd/tinytag", "tags": ["metadata"], "tech": ["audio"]},
+
+  {"id": "py.authomatic", "imports": ["authomatic"], "name": "Authomatic", "repo": "authomatic/authomatic", "tags": ["oauth-openid"], "tech": ["python-web"]},
+  {"id": "py.django-allauth", "imports": ["allauth"], "name": "django-allauth", "repo": "pennersr/django-allauth", "tags": ["auth-toolkit"], "tech": ["python-web"]},
```

Comment:
maybe django-auth for tags and app-arch for tech? not sure, just app-arch/python-web look close with me

Reply:
why do you mean 'look close'? python-web is web applications, it's more specific than 'application architecture'.

Reply:
ignore it, for some reason I thought that some django libs go to app-arch, but others to python-web.